libmcount/dynamic.c

```diff
@@ -26,6 +26,8 @@ static struct mcount_dynamic_stats {
 
 #define PAGE_SIZE   4096
 #define CODE_CHUNK  (PAGE_SIZE * 8)
+#define TRAMPOLINE_SIZE 14
+#define JMP_INSN_SIZE   6
```

Comment:
The word trampoline is too generic and we use it for other places too.  Please be more specific.  Also this file is arch-agnostic please move arch-specific code under arch directory.  You might want to add some weak functions with `mcount_arch_` prefix.